### PR TITLE
Bug Fix | References IE Grid

### PR DESCRIPTION
**Pull request by ChazUK**

### CODE REVIEW

File: src/assets/scss/components/_grid.scss [hunk 1]
```diff
@@ -4,24 +4,20 @@
 }
 
 .govuk__grid-container {
-  grid-template-columns: 33.33% 33.33% auto;
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (omarCreativeDev) [inline src/assets/scss/components/_grid.scss]: what does 1fr mean?

File: src/assets/scss/components/_grid.scss [hunk 2]
```diff
@@ -35,3 +31,41 @@
...
+  grid-column-end: 4;
+}
+
+.govuk-form-group--error {
+  .govuk__grid-column-1 {
+    grid-row-start: 2;
+    grid-row-end: 3;
+  }
+
+  .govuk__grid-column-2 {
```
Comment (omarCreativeDev) [inline src/assets/scss/components/_grid.scss]: you can probs group .govuk__grid-column-1, govuk__grid-column-2 and govuk__grid-column-3 styles under one block

File: src/assets/scss/components/_grid.scss (hunk 1 above)
Comment (ChazUK) [reply src/assets/scss/components/_grid.scss]: `fr` is a flex width. I tried `auto auto auto` but it didn't seem to like it.

Comment (omarCreativeDev) [reply src/assets/scss/components/_grid.scss]: ah cool, thanks for explaining. css grids is so powerful. yea i initially thought IE didnt like `auto` but that wasn't the case

File: src/assets/scss/components/_grid.scss (hunk 2 above)
Comment (ChazUK) [reply src/assets/scss/components/_grid.scss]: done

**Comment by ChazUK** [comment]
[Staff references page broken in IE](https://trello.com/c/GUGHqOgp/301-staff-references-page-broken-in-ie)

**Comment by ChazUK** [comment]
[(2) Workplace references page broken in IE](https://trello.com/c/abJj6sRR/300-2-workplace-references-page-broken-in-ie)

**Pull request merged**